x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -386,39 +393,55 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
 
   const handleRenderCellValue = useCallback(
     (_props: EuiDataGridCellValueElementProps) => {
-      // https://github.com/elastic/eui/issues/5811
-      const idx = _props.rowIndex - pagination.pageSize * pagination.pageIndex;
-      const alert = alerts[idx];
-      // ecsAlert is needed for security solution
-      const ecsAlert = ecsAlertsData[idx];
-      if (alert) {
-        const data: Array<{ field: string; value: string[] }> = [];
-        Object.entries(alert ?? {}).forEach(([key, value]) => {
-          data.push({ field: key, value: value as string[] });
-        });
-        if (isSystemCell(_props.columnId)) {
-          return (
-            <SystemCellFactory
-              alert={alert}
-              columnId={_props.columnId}
-              isLoading={isLoading || isLoadingCases || isLoadingMaintenanceWindows}
-              cases={cases}
-              maintenanceWindows={maintenanceWindows}
-              showAlertStatusWithFlapping={showAlertStatusWithFlapping}
-              caseAppId={casesConfig?.appId}
-            />
-          );
-        }
+      try {
+        // https://github.com/elastic/eui/issues/5811
+        const idx = _props.rowIndex - pagination.pageSize * pagination.pageIndex;
+        const alert = alerts[idx];
+        // ecsAlert is needed for security solution
+        const ecsAlert = ecsAlertsData[idx];
+        if (alert) {
+          const data: Array<{ field: string; value: string[] }> = [];
+          Object.entries(alert ?? {}).forEach(([key, value]) => {
+            data.push({ field: key, value: value as string[] });
+          });
+          if (isSystemCell(_props.columnId)) {
+            return (
+              <SystemCellFactory
+                alert={alert}
+                columnId={_props.columnId}
+                isLoading={isLoading || isLoadingCases || isLoadingMaintenanceWindows}
+                cases={cases}
+                maintenanceWindows={maintenanceWindows}
+                showAlertStatusWithFlapping={showAlertStatusWithFlapping}
+                caseAppId={casesConfig?.appId}
+              />
+            );
+          }
 
-        return renderCellValue({
-          ..._props,
-          data,
-          ecsData: ecsAlert,
-        });
-      } else if (isLoading) {
-        return <EuiSkeletonText lines={1} />;
+          return renderCellValue({
+            ..._props,
+            data,
+            ecsData: ecsAlert,
+          });
+        } else if (isLoading) {
+          return <EuiSkeletonText lines={1} />;
+        }
+        return null;
+      } catch (e) {
```

Comment:
should we surface this error somehow to the user? Maybe have something that can be expanded and show the actual error? @XavierM thoughts?

Reply:
I asked @umbopepato to do that because we had SDH before when we do not control what the user of the table are doing. 
Yes you are right!, it will be good to use an expand button of the cell or an icon to show the full detailed of the message.

Reply:
Added a code block containing the error stack that is only visible in the cell popover